.circleci/config.yml

```diff
@@ -22,6 +22,7 @@ jobs:
             - v1-dependencies-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}-
 
       - run: ./gradlew dependencies
+      - run: bash <(curl -s https://codecov.io/bash)
```

Comment:
@ugoamanoh this should be run after the tests, no?

Reply:
Yes, 
This is still work in progress but I had to push to test if codecov reports show on the PR

Reply:
I see what you mean. Thanks
This is still WIP